stan-dev/rstan
# Stan headers 2.26

#### Summary:

Merge the work that culminated in StanHeaders 2.26.27 on CRAN.

#### Intended Effect:

Bring the develop branch up to date

#### How to Verify:

It has already passed `R CMD check` on CRAN and does not have any merge conflicts currently, but this PR will cause the unit tests to be run again.

#### Side Effects:

None of the effects are "side", but this puts develop into an intermediate state with 2.26 libraries that have been heavily patched to be backwards compatible with 2.21. 

#### Documentation:

No changes to the docs because StanHeaders has little documentation of its own and it is intended to be backward compatible. There were some tweaks to one of the vignettes that describes how to use Stan Math directly rather than through RStan.

#### Reviewer Suggestions: 

Not needed. This has already been reviewed.

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company): Columbia University

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: GPLv3 (http://opensource.org/licenses/GPL-3.0)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
